### PR TITLE
fix show-text-modal-directive.js(#4131)

### LINKED ISSUE #4131: show-text-modal 特殊json值显示错误
- [x] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [x] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [x] 我已经仔细检查过[FAQ](https://www.apolloconfig.com/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**

配置值为json类型，其值包含大于或等于16位数字时，会在原值后加入该数字

**复现**

通过如下步骤可以复现：
配置：
json：
{
    "id": 4,
    "partnerShopLabelList": "MDBQ201911040002000001",
    "partnerOrderUnitChangeCount": 10000
}
image

现象：
image

**期望**
不追加后缀

_**此问题应为 https://github.com/apolloconfig/apollo/pull/3851 _引入**_
image

**截图**

如果可以，附上截图来描述你的问题

### 额外的细节和日志

- 版本：
- 错误日志
- 配置：
- 平台和操作系统

### PR DESCRIPTION
**Pull request by CalebZYC**
## What's the purpose of this PR
fix #4131

case1
```
{ "ext1": "||390123456000555648889990|", "ext2": "====++++++++++hhhhhhhhhhhhhhh00|||", "ext3": 10000111111111111111111111111111111111111, "ext4":"||0123456789|", "ext5":"123||_+++||--", "ext6":231313, "ext7":"|@^(!)---++++123456789#***||&&&*#|" }
```
image

case2
```
[9587743934784070, 132469186976563400000000000000000000000000005, 127525757991925097]
```
image

case3
```
["|||9587743934784070||", 132469186976563450000000000000000000000, 127525757991925097,"|hhhhhh||", "|||", "|12313|999===|", "|1239088888888888888888888886||"]
```
image

case4
```
{"ext1":"||111111111111111111111111111111#*0}}|", "ext2":"|00001|||||000||"}
```
image

case5
```
{ "id": 4, "label": "MBG1011110000000000000000000000000001", "count": 1002222222222222222222222222222222200, "ext1":"|100000000000000000000000000000000|", "ext2":"||111111111111111111111111111111#}}|", "ext3":"||1111111111111111111111110000|||"}
```
image

case6
```
["MDBQ2019110400021111111110000011111111111", "||201911040002111111111000001|||",11111111111111111111111111111111111111111,"BBBBB", 999111233333333333333333333333333333, "|00||==__+////\\\\||"]
```
image

case7
```
111111112121211111111111111111111111111111111111111
```
image

case8
```
{ "id": 4, "partnerShopLabelList": "MDBQ201911040002000001", "partnerOrderUnitChangeCount": 10000 }
```
image

## Which issue(s) this PR fixes:
Fixes #4131

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [ ] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [ ] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [ ] Update the [`CHANGES` log](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by nobodyiam** [comment]
As this is a bug fix, would you please also update the CHANGES.md?

**Comment by CalebZYC** [comment]
> As this is a bug fix, would you please also update the CHANGES.md?

Done

File: CHANGES.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Apollo 2.0.0
 * [Fix the issue that property placeholder doesn't work for dubbo reference beans](https://github.com/apolloconfig/apollo/pull/4175)
 * [Fix the NPE occurred when using EnableApolloConfig with Spring 3.1.1](https://github.com/apolloconfig/apollo/pull/4180)
 * [Bump guava from 29.0 to 31.0.1](https://github.com/apolloconfig/apollo/pull/4182)
+* [fix show-text-modal-directive.js](https://github.com/apolloconfig/apollo/pull/4183)
```
Comment (nobodyiam) [inline CHANGES.md:33]:
```suggestion
* [fix the json number display issue when it's longer than 16](https://github.com/apolloconfig/apollo/pull/4183)
```

**Review by nobodyiam — APPROVED** [review]
lgtm

**Pull request merged**